azure/package.json

```diff
@@ -32,33 +32,26 @@
 		"clean:docs": "rimraf **/_api-extractor-temp docs/api/*/**",
 		"clean:nyc": "rimraf nyc/**",
 		"format": "npm run prettier:fix",
-		"postinstall": "npm run postinstall:lerna",
+		"preinstall": "node ../scripts/only-pnpm.cjs",
 		"layer-check": "fluid-layer-check --info ../build-tools/packages/build-tools/data/layerInfo.json",
 		"lerna": "lerna",
-		"lint": "lerna run lint --no-sort --stream -- -- -- --color",
-		"lint:fix": "lerna run lint:fix --no-sort --stream -- -- -- --color",
+		"lint": "lerna run lint --no-sort --stream",
+		"lint:fix": "lerna run lint:fix --no-sort --stream",
 		"policy-check": "cd .. && npm run policy-check",
 		"policy-check:asserts": "cd .. && npm run policy-check:asserts",
 		"policy-check:fix": "cd .. && npm run policy-check:fix",
-		"post-lerna-install:default": "[ -f package-lock.json ] && mv package-lock.json lerna-package-lock.json; [ -f package-lock.json.bak ] && mv package-lock.json.bak package-lock.json; exit 0",
-		"post-lerna-install:win32": "if exist package-lock.json move package-lock.json lerna-package-lock.json && if exist package-lock.json.bak move package-lock.json.bak package-lock.json",
-		"postinstall:lerna": "run-script-os",
-		"postinstall:lerna:default": "npm run pre-lerna-install:default && { lerna bootstrap --strict --hoist;t1=$?;npm run post-lerna-install:default;exit $t1;}",
-		"postinstall:lerna:win32": "npm run pre-lerna-install:win32 && (lerna bootstrap --strict --hoist & npm run post-lerna-install:win32)",
-		"pre-lerna-install:default": "[ -f package-lock.json ] && mv package-lock.json package-lock.json.bak; [ -f lerna-package-lock.json ] && mv lerna-package-lock.json package-lock.json; exit 0",
-		"pre-lerna-install:win32": "if exist package-lock.json move package-lock.json package-lock.json.bak && if exist lerna-package-lock.json move lerna-package-lock.json package-lock.json",
```

Comment:
Can you shortly explain why this was removed? (as far as I remember, this made sense on the top level anyways, not in azure here)